xml/cap_admin_certificates.xml

```diff
@@ -355,4 +355,45 @@
 --version &latestscfchart;
 </screen>
  </sect1>
+ <sect1 xml:id="sec-cap-trusted-certs">
+   <title>Difference between TRUSTED_CERTS and ROOTFS_TRUSTED_CERTS</title>
+   <para>
+     The <literal>TRUSTED_CERTS</literal> parameter contains a list of CA certificates which become
+     available in application instance containers as files in the directory
+     <filename>/etc/cf-system-certificates</filename> and the general trust store
+     (Only cflinuxfs3).
+     The enviromental value <literal>CF_SYSTEM_CERT_PATH</literal> holds the first path.
```

Comment:
Missing an 'n' in `environmental value`. However, I would suggest using `environment variable` instead since that's more common in the CF ecosystem.